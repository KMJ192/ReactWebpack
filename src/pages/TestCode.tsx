import TestCodeComponent from '@src/components/TestCodeComponent';
import React, { useState } from 'react';

function TestCode() {
  const [count, setCount] = useState(0);

  const increase = () => {
    setCount(count + 1);
  };

  const decrease = () => {
    setCount(count - 1);
  };

  return (
    <>
      {/* <TestCodeComponent count={count} increase={increase} decrease={decrease} /> */}
      <a href='https://naver.com'>counter</a>
    </>
  );
}

export default TestCode;
