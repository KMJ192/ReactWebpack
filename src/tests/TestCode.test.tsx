import React from 'react';
import { render } from '@testing-library/react';

import TestCode from '../pages/TestCode';

describe('', () => {
  const component = render(<TestCode />);
  expect(component.container).toMatchSnapshot();
});
