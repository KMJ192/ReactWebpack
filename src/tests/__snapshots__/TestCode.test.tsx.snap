// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <div>
    <h1>
      Counter
    </h1>
    <h2>
      0
    </h2>
    <button>
      증가
    </button>
    <button>
      감소
    </button>
  </div>
</div>
`;
